src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const PersonAge = () => (
+    age
+      ? <p className="Person__age">{`I am ${age}`}</p>
+      : ''
+  );
```

Comment:
Why do you need a lot of components?
Code written using JSX will be converted into React.createElement() calls.
Just do it like variable
```suggestion
  const personAge = age && <p className="Person__age">{`I am ${age}`}</p>;
```